bool recu_unidriver_got_msg_from_sp
bool got_cmd_connect_to_tool
bool got_cmd_disconect_from_tool
bool got_cmd_release_lf
bool got_cmd_grab_lf

bool recu_unidriver_got_msg_from_recu
bool robot_not_connected_to_tool
bool robot_connected_to_lf_tool
bool robot_connected_to_atlas_tool
bool robot_connected_to_filter_tool
bool undefined_connection_detected
bool robot_tool_connection_failure
bool ladder_frame_not_connected
bool ladder_frame_connected
bool ladder_frame_connection_failure
bool pressure_ok
bool pressure_failure
